packages/kamino-sdk/README.md

```diff
@@ -169,4 +169,69 @@ tx = await assignBlockInfoToTransaction(connection, tx, signer.publicKey);
 const txHash = await sendAndConfirmTransaction(connection, tx, [signer], {
   commitment: 'confirmed',
 });
-```
\ No newline at end of file
+```
+
+### Deposit shares
+
+Deposit custom amount of token A and B for a specific strategy, example code:
+
+```javascript
+import { clusterApiUrl, Connection, PublicKey, sendAndConfirmTransaction, Keypair, Transaction } from '@solana/web3.js';
+import { 
+  Kamino,
+  getAssociatedTokenAddressAndData,
+  createTransactionWithExtraBudget,
+  getCreateAssociatedTokenAccountInstructionsIfNotExist,
+  assignBlockInfoToTransaction
+} from '@hubbleprotocol/kamino-sdk';
+import Decimal from 'decimal.js';
+
+// setup Kamino SDK
+const strategyPubkey = new PublicKey('2H4xebnp2M9JYgPPfUw58uUQahWF8f1YTNxwwtmdqVYV'); // you may also fetch strategies from hubble config
+const owner = new PublicKey('HrwbdQYwSnAyVpVHuGQ661HiNbWmGjDp5DdDR9YMw7Bu'); // wallet with shares
```

Comment:
why does the wallet need to have shares already?

Reply:
Ah, I copied the sample from withdraw function, it doesnt need existing shares